dev-python/imapclient/imapclient-2.0.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} pypy )
```

Comment:
Can you try to add pypy3 if it's supported?

Reply:
Can't test. pypy3 isn't supported for my arch (ppc64).

Reply:
That's fine.